.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "✨ Feature request"
+about: Suggest a new feature or idea
+title: ''
+labels: "feature request, awaiting triage"
+assignees: ''
+
+---
+
+<!--
+
+  Before you raise this feature request, please check that it isn’t already captured in:
+  - the community backlog: https://design-system.service.gov.uk/community/backlog/
+  - an existing Github issue: https://github.com/alphagov/govuk-frontend/issues
+
+  If you are proposing a new component or pattern, please follow the instructions here: https://design-system.service.gov.uk/community/propose-a-component-or-pattern/
+
+  Please fill in as much of the template below as you’re able to. If you're unsure whether the issue already exists or how to fill in the template, open an issue anyway. Our team will help you to complete the rest.
+
+-->
+
+## Related component
+<!-- Does this feature relate to an existing component? -->
+
+## Context
+<!-- What are you trying to do? Is this something you think should behave differently, or something that you currently cannot do? Is this related to an existing issue/bug? -->
+
+## Alternatives
+<!-- Are you currently using a workaround / alternative solution instead? -->
+
```

Comment:
Just an idea but I wonder if we should add one more section to ask whether the person raising this thinks that any documentation for the feature might be required. Perhaps especially in terms of whether the feature will need guidance to be added to the Design System website? 

The section name could be followed by "(if applicable)" as we can't expect users to necessarily be aware of how we document things but it would be useful insight if they had any thoughts in this area.

Reply:
@hannalaakso That's an interesting idea - can you give an example of the kind of thing a user might note down in this section, so I get a better idea of what you mean?

Reply:
For instance, with a feature request such as https://github.com/alphagov/govuk-frontend/issues/1656 it would be interesting to understand how the person raising the issue sees the feature being documented in the Design System. This particular feature request touches on all the components and other things too, so it might not be immediately clear to our team where/how we would tell our users about it. Asking questions about documentation could possibly prompt the person raising the card to expand on whether the feature they've proposed would be opt-in or on by default too. 

But I'm happy to leave this out at least for now, we'd probably want to think about it a bit more..! Maybe once we have a better idea of how the issue templates are received we can think of further tweaks like this.

Reply:
Thanks @hannalaakso , really helpful to see that issue! From my point of view, I think I'd prefer to leave that out for now (at least for the trial) and see how things go... My gut instinct is that for the majority of issues raised it'll be clear how any change affects our documentation and it'll be something we can deal with internally as a team (I guess that's something we can test during the trial), so I'm leaning towards leaving that section out and either expecting more advanced users to pro-actively add details if they think it requires a big documentation change __or__ we have to go back to the user and ask.